tests/testthat/test-paginate_listing.R

```diff
@@ -0,0 +1,33 @@
+testthat::test_that("pagination works vertically", {
+  # pre-processing and ordering
+  tmp_data <- formatters::ex_adae %>%
+    dplyr::select(USUBJID, AGE, BMRKR1) %>%
```

Comment:
If this is the only place we use dplyr, we should consider doing this via base R to drop the dependency.

Reply:
It makes sense and as it is, `rlistings` can drop this dependency. Still, the roadmap specifies that users should pre-process the data considerably before using `rlistings`. I think the majority of people would use `dplyr` for this, or have it installed on their machines. Also, I believe that with a growing number of listings needed, it may become better to use `dplyr` directly instead of finding ways to use `base` to do the same